sklearn/tree/_sorting.pyx

```diff
@@ -0,0 +1,120 @@
+from libc.math cimport log2
+
+
+def _py_sort(float32_t[::1] feature_values, intp_t[::1] samples, intp_t n):
```

Comment:
Is this function just moved? Is there a reason for this move? I think not moving would make the diff easier to review.